samples/sh/hello-world/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/PowerShell/DSC-Samples/bash/resources/hello-world
+
+go 1.19
+
+require (
+	github.com/TylerBrock/colorjson v0.0.0-20200706003622-8a50f05110d2 // indirect
+	github.com/fatih/color v1.15.0 // indirect
+	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/knadh/koanf/maps v0.1.1 // indirect
+	github.com/mattn/go-colorable v0.1.13 // indirect
+	github.com/mattn/go-isatty v0.0.17 // indirect
+	github.com/mitchellh/copystructure v1.2.0 // indirect
+	github.com/mitchellh/reflectwalk v1.0.2 // indirect
+	github.com/spf13/cobra v1.7.0 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/thediveo/enumflag v0.10.1 // indirect
+	golang.org/x/sys v0.6.0 // indirect
+)
```

Comment:
This isn't needed - no dependencies for the project, we just need this for the docs site.
```suggestion
```